scripts/src/main/resources/scripts/command/az

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+function doSetup() {
+  if command -v az &> /dev/null
+  then
+    if [ "${1}" != "silent" ] && ! doIsQuiet
+    then
+      doEcho "Azure CLI is already installed at $(command -v az)"
+    fi
+  else
+    doRequireNotBatch
+    if doIsWindows
+    then
+      # Get leatest release
+      if [ -z "${AZURECLI_VERSION}" ]
+      then
+        doEcho "Getting latest release..."
+        AZURECLI_VERSION=$(curl -s https://api.github.com/repos/Azure/azure-cli/releases/latest | awk -F ":" '/tag_name/ {print $2}'| awk -F "\"" '{print $2}' | awk -F "-" '{print $3}')
+      fi
+      local version="${AZURECLI_VERSION}"
+      doDownload "-" "" "az" "${version}"
+      # Install Azure CLI for Windows
+      doEcho "Installing Azure CLI for Windows..."
+      installOnWindows
+    else
+      doFail "Sorry, Azure CLI installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
+    fi
+  fi
+}
+
+# Install Azure CLI for Windows via Powershell Command
+function installOnWindows() { 
+  winPath=$(cygpath -w "${DEVON_DOWNLOAD_DIR}")
```

Comment:
this is a global variable. Better do
```
local winPath
winPath=$(cygpath -w "${DEVON_DOWNLOAD_DIR}")
```

As the variable is only used in one place, you can even omit the variable.
However, from a documentation point of view the variable makes it more readable instead of putting `$(cygpath -w "${DEVON_DOWNLOAD_DIR}")` directly instead of `${winPath}` so it is fine to keep the variable.